### PR TITLE
Add blog post to announce Cucumber-JVM v2.0.0.

### PR DESCRIPTION
**Pull request by brasmusson**

### CODE REVIEW

File: apps/dynamic/views/_posts/announcing-cucumber-jvm-2-0-0.md [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+- `EmbedEvent` - calling `scenario.embed` in a hook triggers this event.
+- `WriteEvent` - calling `scenario.write` in a hook triggers this event.
+- `TestStepFinished` - sent after the execution of a Test Step, contains the Test Step and its Result.
+- `TestCaseFinished` - sent after the execution of a Test Case(/Pickle/Scenario), contains the Test Case and its Result.
+- `TestRunFinished` - the last event sent.
+
+#### Java8
+The improvements with respect to Java8 inlucdes:
+- method references can be used in lambda step definitions
+- lambdea steps are allowed to throw checked exceptions.
```
Comment (mpkorstanje) [inline apps/dynamic/views/_posts/announcing-cucumber-jvm-2-0-0.md]: lambdea -> lambda

File: apps/dynamic/views/_posts/announcing-cucumber-jvm-2-0-0.md [hunk 2]
```diff
@@ -0,0 +1,82 @@
+---
+layout: post
+nav: blog
+date: "2017-08-20 09:00:00"
+title: "Announcing Cucumber-JVM v2.0.0"
+author: Björn Rasmusson
+---
+
+I am excited to announce the release of Cucumber-JVM v2.0.0. Below follows a summary of (some) notable changes, and the non backward compatible changes. The full change log is available [here](https://github.com/cucumber/cucumber-jvm/blob/master/History.md). Most importantly do not miss that the Maven group id is changed.
```
Comment (mlvandijk) [inline apps/dynamic/views/_posts/announcing-cucumber-jvm-2-0-0.md]: Suggestion: "Most importantly do not miss that the Maven group id is changed. " -> "Most importantly note that the Maven group id is changed. "

File: apps/dynamic/views/_posts/announcing-cucumber-jvm-2-0-0.md [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+#### Maven group id
+The Maven group id is changed to `io.cucumber`.
+
+#### Gherkin v4 and Pickles
+The Gherkin library is updated from version 2.12.2 to version 4.1.3. This is
+a complete rewrite using new design principles. One of the more visible
+consequenses for Cucumber-JVM is that the feature files are compiled into a
+set of [Pickles](https://github.com/cucumber/cucumber/tree/master/gherkin#pickles) (one Pickle for each Scenario and one for each Example Table row),
+which form the basis for the execution. Before the actual execution Pickle are
+converted to Test Case, by matching the steps to step definitions and adding
```
Comment (mlvandijk) [inline apps/dynamic/views/_posts/announcing-cucumber-jvm-2-0-0.md]: Suggestion: "Before the actual execution Pickle are
+converted to Test Case" -> "Before the actual execution a Pickle is
+converted to a Test Case

File: apps/dynamic/views/_posts/announcing-cucumber-jvm-2-0-0.md [hunk 4]
```diff
@@ -0,0 +1,82 @@
...
+steps for the hooks. Previosly the execution was based on iterating over the
```
Comment (mlvandijk) [inline apps/dynamic/views/_posts/announcing-cucumber-jvm-2-0-0.md]: Previosly -> Previously

File: apps/dynamic/views/_posts/announcing-cucumber-jvm-2-0-0.md [hunk 5]
```diff
@@ -0,0 +1,82 @@
...
+- `TestCaseStarted` - sent before starting the execution of a Test Case(/Pickle/Scenario), contains the Test Case
+- `TestStepStarted` - sent before starting the execution of a Test Step, contains the Test Step
+- `EmbedEvent` - calling `scenario.embed` in a hook triggers this event.
+- `WriteEvent` - calling `scenario.write` in a hook triggers this event.
+- `TestStepFinished` - sent after the execution of a Test Step, contains the Test Step and its Result.
+- `TestCaseFinished` - sent after the execution of a Test Case(/Pickle/Scenario), contains the Test Case and its Result.
+- `TestRunFinished` - the last event sent.
+
+#### Java8
+The improvements with respect to Java8 inlucdes:
```
Comment (mlvandijk) [inline apps/dynamic/views/_posts/announcing-cucumber-jvm-2-0-0.md]: inlucdes -> includes

File: apps/dynamic/views/_posts/announcing-cucumber-jvm-2-0-0.md [hunk 6]
```diff
@@ -0,0 +1,82 @@
...
+- method references can be used in lambda step definitions
+- lambdea steps are allowed to throw checked exceptions.
+
```
Comment (mlvandijk) [inline apps/dynamic/views/_posts/announcing-cucumber-jvm-2-0-0.md]: Suggest to add the following: "It is no longer possible to use lambda step definitions without also using cucumber-java8"

File: apps/dynamic/views/_posts/announcing-cucumber-jvm-2-0-0.md [hunk 7]
```diff
@@ -0,0 +1,82 @@
...
+- `TestStepStarted` - sent before starting the execution of a Test Step, contains the Test Step
+- `EmbedEvent` - calling `scenario.embed` in a hook triggers this event.
+- `WriteEvent` - calling `scenario.write` in a hook triggers this event.
+- `TestStepFinished` - sent after the execution of a Test Step, contains the Test Step and its Result.
+- `TestCaseFinished` - sent after the execution of a Test Case(/Pickle/Scenario), contains the Test Case and its Result.
+- `TestRunFinished` - the last event sent.
+
+#### Java8
+The improvements with respect to Java8 inlucdes:
+- method references can be used in lambda step definitions
```
Comment (mlvandijk) [inline apps/dynamic/views/_posts/announcing-cucumber-jvm-2-0-0.md]: . at end of sentence is missing

File: apps/dynamic/views/_posts/announcing-cucumber-jvm-2-0-0.md [hunk 8]
```diff
@@ -0,0 +1,82 @@
...
+Now the `TestStarted` notification is always fired before the test is started, so the `--junit,--allow-started-ignored` option has no effect and is deprecated. This change in notifications is enabled by using the `AssumptionFailed` notification to make Undefined and Pending Scenarios be marked as skipped.
+
+#### TestNG module
+Now it is the Test Cases(/Pickles/Scenarios) that are mapped to TestNG tests.
+
+#### Tag Expressions
+Now the [Tag Expression library](https://github.com/cucumber/cucumber/tree/master/tag-expressions) is used to parse the `--tags` options and the tags conditions on hooks. Therefore expressions like `--tags 'not @foo or @bar'` or `Before("@foo and @bar")` can be used. The old syntax for "not" (`~@foo`) and "or"  (`@foo,@bar`) are deprecated.
+
+#### Tag, name, and line filters can coexist
+For instance `--tags @foo classpath:path/file.feature:3` is possible to be used. The different filters are combined with "and" (all must be fulfilled). One practical use of this is when the test run that generated a rerun file was using a `--tags` option, then that `--tags` option does not need to be removed when using the rerun file, which previosly was the case.
```
Comment (mlvandijk) [inline apps/dynamic/views/_posts/announcing-cucumber-jvm-2-0-0.md]: Suggestion & typo: "For instance `--tags @foo classpath:path/file.feature:3` is possible to be used. The different filters are combined with "and" (all must be fulfilled). One practical use of this is when the test run that generated a rerun file was using a `--tags` option, then that `--tags` option does not need to be removed when using the rerun file, which previosly was the case. " ->
"For instance it is possible to use`--tags @foo classpath:path/file.feature:3`. The different filters are combined with "and" (all must be fulfilled). One practical use of this is when the test run that generated a rerun file was using a `--tags` option, then that `--tags` option does not need to be removed when using the rerun file, which previously was the case.

File: apps/dynamic/views/_posts/announcing-cucumber-jvm-2-0-0.md [hunk 9]
```diff
@@ -0,0 +1,82 @@
...
+Now it is the Test Cases(/Pickles/Scenarios) that are mapped to TestNG tests.
+
+#### Tag Expressions
+Now the [Tag Expression library](https://github.com/cucumber/cucumber/tree/master/tag-expressions) is used to parse the `--tags` options and the tags conditions on hooks. Therefore expressions like `--tags 'not @foo or @bar'` or `Before("@foo and @bar")` can be used. The old syntax for "not" (`~@foo`) and "or"  (`@foo,@bar`) are deprecated.
+
+#### Tag, name, and line filters can coexist
+For instance `--tags @foo classpath:path/file.feature:3` is possible to be used. The different filters are combined with "and" (all must be fulfilled). One practical use of this is when the test run that generated a rerun file was using a `--tags` option, then that `--tags` option does not need to be removed when using the rerun file, which previosly was the case. 
+
+#### Ambiguous as a separate result type
+Preiously ambiguous step (more that one step definition matches the step) were using the `failed` result type. Now we follow the lead of Cucumber-JS and use a separate `ambiguous` result type for ambiguouos steps. A summary like `2 Scenarios (1 Passed, 1 Ambiguous)` may now occur.
```
Comment (mlvandijk) [inline apps/dynamic/views/_posts/announcing-cucumber-jvm-2-0-0.md]: Preiously -> Previously
ambiguouos -> ambiguous

File: apps/dynamic/views/_posts/announcing-cucumber-jvm-2-0-0.md [hunk 10]
```diff
@@ -0,0 +1,82 @@
...
+
+Non backward compatible changes 
+-------------------------------
+
+#### Formatter interface
+Formatters are now Event Listeners that listens to events. The formatter register itself as a handler for the events it needs to listen to, and then process the events received to produce its output. See the "Event Model" section above for a listing of the available events, and for instance the [source code of the Json Formatter](https://github.com/cucumber/cucumber-jvm/blob/master/core/src/main/java/cucumber/runtime/formatter/JSONFormatter.java) for an example
```
Comment (mlvandijk) [inline apps/dynamic/views/_posts/announcing-cucumber-jvm-2-0-0.md]: listens -> listen
register -> registers
process -> processes
. missing at end of sentence

File: apps/dynamic/views/_posts/announcing-cucumber-jvm-2-0-0.md [hunk 11]
```diff
@@ -0,0 +1,82 @@
...
+
+#### Data available to hooks
+Two of the methods in the `Scenario` object available to hooks have change.
```
Comment (mlvandijk) [inline apps/dynamic/views/_posts/announcing-cucumber-jvm-2-0-0.md]: change -> changed

File: apps/dynamic/views/_posts/announcing-cucumber-jvm-2-0-0.md [hunk 12]
```diff
@@ -0,0 +1,82 @@
...
+
+The `getStatus` method does now return an Enum instead of a String. Using `scenario.getStatus().lowerCaseName()` the same string that previously was return can be obtained. To check what the status value is statements like `scenario.getStatus() == Result.Type.FAILED` can be used.
```
Comment (mlvandijk) [inline apps/dynamic/views/_posts/announcing-cucumber-jvm-2-0-0.md]: return -> returned

Suggestions: 
"does now return" -> now returns
add comma between "is" and "statements" -> "is, statements"

File: apps/dynamic/views/_posts/announcing-cucumber-jvm-2-0-0.md [hunk 13]
```diff
@@ -0,0 +1,82 @@
...
+
+#### Formatter interface
+Formatters are now Event Listeners that listens to events. The formatter register itself as a handler for the events it needs to listen to, and then process the events received to produce its output. See the "Event Model" section above for a listing of the available events, and for instance the [source code of the Json Formatter](https://github.com/cucumber/cucumber-jvm/blob/master/core/src/main/java/cucumber/runtime/formatter/JSONFormatter.java) for an example
+
+#### Data available to hooks
+Two of the methods in the `Scenario` object available to hooks have change.
+
+The `getStatus` method does now return an Enum instead of a String. Using `scenario.getStatus().lowerCaseName()` the same string that previously was return can be obtained. To check what the status value is statements like `scenario.getStatus() == Result.Type.FAILED` can be used. 
+
+The `getId` method does now return a string `<feature file uri>:<line number>`. Now when Pickes are the basis for the test execution the data used previously (like "feature id", "examples id") is no longer available during the execution.
```
Comment (mlvandijk) [inline apps/dynamic/views/_posts/announcing-cucumber-jvm-2-0-0.md]: Suggestions: 
"does now return" -> now returns
add comma between "execution" and "the" -> "execution, the"

**Review by mlvandijk — COMMENTED** [review]
Added my suggestions inline

**Pull request merged**